src/main/java/games/strategy/engine/framework/GameRunner.java

```diff
@@ -181,6 +183,7 @@ public static void main(final String[] args) {
     showMainFrame();
     new Thread(() -> setupLogging(GameMode.SWING_CLIENT)).start();
     HttpProxy.setupProxies();
+    checkLocalSystem();
```

Comment:
I restored the local system check (albeit extracted to a new private method) in approximately the same place where it was removed a year ago.  I also did what the previous code did and run it synchronously.  Even if the network check is delayed due to latency issues, it doesn't seem to delay the appearance of the main screen because the check is running on the main thread after the EDT has already spun up (`showMainFrame()` invoked three lines prior).

However, it may delay the remaining statements in this method.  Should it be run on a background thread like `checkForUpdates()` is?